test/realizations/catchments/LSTM_Realization_Test.cpp

```diff
@@ -2,7 +2,7 @@
 
 #include "gtest/gtest.h"
 #include "realizations/catchment/LSTM_Realization.hpp"
-#include "forcing/Forcing.h"
+//#include "forcing/Forcing.h"
```

Comment:
Note that these tests are almost certainly not actually being run because of the guard at line 1...I am betting that if we can run these tests they won't succeed with only this change.

Reply:
At the moment, Ngen framework does not seem to set up to handle LSTM module. So there is actually no test code ever built because of the guard lines.